src/Pages/Templates/TemplatesTable/TemplatesTable.tsx

```diff
@@ -250,22 +280,26 @@ const TemplatesTable = () => {
             >
               <Thead>
                 <Tr>
-                  {columnHeaders.map(({ title, width }, index) => (
-                    <Th key={title + 'column'} width={width} sort={sortParams(index)}>
-                      {title}
-                    </Th>
-                  ))}
+                  {columnHeaders.map(({ title, width }, index) =>
+                    title === 'Status' ? (
+                      <Th key={title + 'column'} width={width}>
+                        {title}
+                      </Th>
+                    ) : (
+                      <Th key={title + 'column'} width={width} sort={sortParams(index)}>
+                        {title}
+                      </Th>
+                    ),
+                  )}
                   <Th className={classes.spinnerMinWidth}>
                     <Spinner size='md' className={actionTakingPlace ? '' : classes.invisible} />
                   </Th>
                 </Tr>
               </Thead>
               <Tbody>
-                {templateList.map(
-                  (
-                    { uuid, name, description, arch, version, date, use_latest }: TemplateItem,
-                    index,
-                  ) => (
+                {templateList.map((rowData: TemplateItem, index) => {
+                  const { uuid, name, description, arch, version, date, use_latest } = rowData;
```

Comment:
#nit: I think you could keep the simplified direct return here. 
No need to destructure from the variable.

Reply:
fix pushed

Reply:
I see what you were doing there.. The change looks good, but what you had was likely fine. 😓